### ISSUE TITLE
Can operator "Copy" stop the grad flow ?

### ISSUE THREAD

**Issue by N-a-g-i** [facebookarchive/caffe2#1915]
Hi, everyone,
I'm just wondering whether the operator "Copy" is to some extent similar to

       <new>Variable =  <old>Variable.detach()

in PyTorch or not.  More specifically, given the code block below:

      model.net.SubNet1("features", "out_branch1")
      model.net.Copy("features", "features_")
      model.net.SubNet2("features_", "out_branch2")
and we have:
     
      model.net.Loss1("out_branch1", "loss1")
      model.net.Loss2("out_branch2", "loss2")
Then can the "Copy" op stop the grads of "loss2" from propagating down through the backbone net ( as we use detach() in PyTorch ) ?  
If not, how can I do this in Caffe2 ?
